CMakeLists.txt

```diff
@@ -200,6 +200,7 @@ else ()
     # Faasm SGX support
     if (FAASM_SGX_SUPPORT)
         add_subdirectory(src/sgx)
+        add_subdirectory(src/sgx-util)
```

Comment:
It appears that most of the `sgx-util` directory is a copy of the Faabric utils directory. It seems that the additional content is also not specific to SGX. As a result, it should probably live in a new namespace called `faasm::util`, which would live at `faasm/src/util`. 

There are some parts of Faabric that should also be moved into `faasm/src/util`, for example the elements of the `SystemConfig` class that are specific to Faasm, but this is another story.

Reply:
I cleaned the directory up :heavy_check_mark:  But i did not create a new namespace